pycontrails/datalib/ecmwf/arco_era5.py

```diff
@@ -0,0 +1,563 @@
+"""Support for `ARCO ERA5 <https://cloud.google.com/storage/docs/public-datasets/era5>`_.
+
+This module supports:
+
+- Downloading ARCO ERA5 model level data for specific times and pressure level variables.
+- Downloading ARCO ERA5 single level data for specific times and single level variables.
+- Interpolating model level data to a target lat-lon grid and pressure levels.
+- Local caching of the downloaded and interpolated data as netCDF files.
+- Opening cached data as a :class:`pycontrails.MetDataset` object.
+
+This module requires the following additional dependencies:
+
+- `metview (binaries and python bindings) <https://metview.readthedocs.io/en/latest/python.html>`_
+- `lxml` <https://lxml.de/>
+- `gcsfs <https://gcsfs.readthedocs.io/en/latest/>`_
```

Comment:
Same with this?

Should we create some optional depenency `arco-era5` with these?

Obviously `metview` has to stay here.

Reply:
Now `gcsfs` is part of the `zarr` dependency: a1999f460e922a217b341cff909381939c9f48eb